integration/fuse/bin/alluxio-fuse

```diff
@@ -101,6 +101,15 @@ fuse_stat() {
     echo -e "$(ps aux | grep [A]lluxioFuse | awk -F' ' '{print $2 "\t" $(NF-2) "\t" $NF}')"
     return 0
   fi
+  fuse_info=$(ps aux | grep [A]lluxioWorker | grep alluxio\.worker\.fuse\.enabled=true)
```

Comment:
Alluxio configuration can be passed in via jvm opts or conf/alluxio-site.properties. In this way, we can find out the alluxio mount point and alluxio path configured via jvm opts but not via conf/alluxio-site.properties. Considering adding support for the site properties in the next PR?

Reply:
I wasn't aware.

Is it possible to get the `conf/alluxio-site.properties` information from ps (proc)? Otherwise we need to parse the `conf/alluxio-site.properties` file.

As an alternative (which will work for all variants), could the FUSE process write its state (mountpoint, pid, alluxio_path) somewhere? This could then be verified against local `mount` and running processes `ps` to ensure it's up and running.